external/llvm-project/mlir/include/mlir/Transforms/OutlinerUtils.h

```diff
@@ -45,22 +45,22 @@ bool isConstantZero(Operation *op);
 
 class Outliner {
 public:
-  Outliner(function_ref<bool(Operation *)> isAnchorOp,
-           function_ref<bool(Operation *)> isLeadingOp,
-           function_ref<bool(Operation *)> isTrailingOp, StringRef outlineTag)
-      : isAnchorOp(isAnchorOp), isLeadingOp(isLeadingOp),
-        isTrailingOp(isTrailingOp), outlineTag(outlineTag) {}
+  Outliner(function_ref<bool(Operation *)> _isAnchorOp,
```

Comment:
I don't think we need the leading underscores here?

Reply:
I like the leading _ for some classes to distinguish between local vars and members.